runner/repositories.go

```diff
@@ -128,6 +126,8 @@ func (r *Runner) DeleteRepository(ctx context.Context, repoID string) error {
 		return runnerErrors.NewBadRequestError("repo has pools defined (%s)", strings.Join(poolIds, ", "))
 	}
 
+	r.poolManagerCtrl.DeleteRepoPoolManager(repo)
```

Comment:
If this fails, we need to know. We may be left with dangling goroutines.

Reply:
I will do the error checking.